worlds/timespinner/__init__.py

```diff
@@ -177,56 +205,57 @@ def create_item(self, name: str) -> Item:
         if not item.advancement:
             return item
 
-        if (name == 'Tablet' or name == 'Library Keycard V') and not self.is_option_enabled("DownloadableItems"):
-            item.classification = ItemClassification.filler
-        elif name == 'Oculus Ring' and not self.is_option_enabled("EyeSpy"):
-            item.classification = ItemClassification.filler
-        elif (name == 'Kobo' or name == 'Merchant Crow') and not self.is_option_enabled("GyreArchives"):
-            item.classification = ItemClassification.filler
-        elif name in {"Timeworn Warp Beacon", "Modern Warp Beacon", "Mysterious Warp Beacon"} \
-                and not self.is_option_enabled("UnchainedKeys"):
-            item.classification = ItemClassification.filler
+        if hasattr(self, 'options'): # self.options is not always available in this method
```

Comment:
This comment shouldn't be true anymore

Reply:
I hope your right... as its not an easy scenario to test